medicines/web/components/search-results/index.tsx

```diff
@@ -187,35 +187,46 @@ const SearchResults = (props: {
   resultCount: number;
   searchTerm: string;
   showingResultsForTerm: string;
+  disclaimerAgree: boolean;
 }) => {
   const [showDisclaimerWarning, setShowDisclaimerWarning] = useSessionStorage(
     'showDisclaimer',
     true,
   );
 
-  const handleOnCheck = (event: ChangeEvent<HTMLInputElement>): void => {
-    if (event.target.checked) {
-      setTimeout(() => setShowDisclaimerWarning(false), 1000);
-    }
+  const {
+    disclaimerAgree,
+    drugs,
+    page,
+    pageSize,
+    resultCount,
+    searchTerm,
+    showingResultsForTerm,
+  } = props;
+
+  const hasDrugs = drugs.length > 0;
+
+  const handleOnDisclaimerAgree = (
+    event: MouseEvent<HTMLButtonElement>,
+  ): void => {
+    event.preventDefault();
+    setTimeout(() => setShowDisclaimerWarning(false), 1000);
```

Comment:
we should revisit whether this is necessary

Reply:
👍